### DISCUSSION TITLE
can it compile fltk-rs example for windows xp?

### DISCUSSION THREAD

**Discussion by jiangyubao** [rust9x/rust#11] [Q&A]
Hello, I want to compile fltk-rs example for windows xp, now I can compile fltk-rs example for windows 7 32 bit, but it can not run on windows xp. I know that fltk is based on c++98, and fltk can compile and run on windows xp. But fltk-rs need rust 1.55, and rust last support version for windows xp is 1.50, so I can't compile fltk-rs to exe and run on windows xp.

I know than rust9x is based rust 1.60, so it can compile fltk-rs example, and the exe should be run on windows xp.

Now I download your rust9x source code, and try to compile.

will you please show me an demo for fltk-rs run on windows xp?

**Comment by seritools** [comment]
The instructions for setting up rust9x can be found [here](https://github.com/rust9x/rust/wiki#compiling-the-compiler-and-standard-library). Please let me know if something is unclear there!

If you set up rust9x with appropriately old (or otherwise WinXP-supporting) msvc libs, it might work. See the [sample](https://github.com/rust9x/rust9x-sample/blob/main/.cargo/config.toml) for one way of setting up the library imports.
